### ISSUE TITLE
Dart Sass complains about usage of / for division in ladda.scss

### ISSUE THREAD

**Issue by selangley-wa** [hakimel/Ladda#109]
I see messages like the below when using ladda in an Angular 12 application with Dart Sass:

```
DEPRECATION WARNING: Using / for division is deprecated and will be removed in Dart Sass 2.0.0.

Recommendation: math.div($spinnerSize, -2)

More info and automated migrator: https://sass-lang.com/d/slash-div

    ╷
116 │         right: $spinnerSize/-2 + 10;
    │                ^^^^^^^^^^^^^^^
    ╵
    node_modules\ladda\css\ladda.scss 
```

Could we follow their recommendation and use math.div() instead?

**Comment by theodorejb** [comment]
@selangley-wa Thank you for reporting the issue. This is now resolved in v2.0.3.

**Comment by Donatello-za** [comment]
This update broke our build process (using node-sass version 7.0.1):

```
Module build failed: 
        right: math.div($spinnerSize, -2) + 10;
                  ^
      Invalid CSS after "...    right: math": expected expression (e.g. 1px, bold), was ".div($spinnerSize, "
      in /home/<redacted>/node_modules/ladda/css/ladda.scss (line 118, column 20)
```

I reverted back to 2.0.2 for now in order to get around the problem.
